docs/source/releasenotes.rst

```diff
@@ -4,6 +4,11 @@ Release notes
 Upcoming release
 ----------------
 
+11.3.0
```

Comment:
I'm not an expert on releases, so I'll need your guidance one more time @osrjv and @mikahanninen . Is it fine if I update the release notes along the changes in one go like so?

Post-merge, I would `% make publish` in _master_ **rpaframework** as `11.3.0`.

Reply:
Usually, changes are put under `Upcoming release` if there is no immediate plan to make one, but if you're releasing them soon anyways this will be fine. It's unavoidable that the release notes and PyPI release will be out of sync for a short while, but let's try to minimize that

Reply:
Assuming this is now tested and ready for release. If there's only a need to get a testable package for VSCode development, it's better to merge under `Upcoming` and distribute through devpi instead of PyPI.